src/documentation/about_home-assistant_yellow.html

```diff
@@ -0,0 +1,36 @@
+---
+layout: system-description
+guideName: About Home Assistant Yellow
+---
+
+<div class="container">
+
+  <h2>The expandable smart home hub for Home Assistant</h2>
+  <span
+    >Home Assistant Yellow is the first smart home hub that was produced specifically for Home Assistant. It can be expanded with additional computing power or storage to grow with your smart home. 
+    With Home Assistant Yellow, you can control your lights, thermostat,
+    TV, music, and more. Home Assistant integrates with over 1,000
+    different devices and services. Home Assistant works with popular
+    brands like Philips Hue and also that odd-ball lightbulb you picked up
+    at a sale.</span>
+
+    <h3>Different flavours</h3>
+    <span
+      >Home Assistant Yellow comes in three different flavours:<ul>
+        <li><b>Home Assistant Yellow</b>: The version that is ready to go out of the box. It comes with everything pre-installed, including the Home Assistant operating system.</li>
```

Comment:
```suggestion
        <li><b>Home Assistant Yellow</b>: The version that is ready to go out of the box. It comes with everything pre-installed, including the Home Assistant operating system. This Kit does not come with an external power supply.</li>
```

Reply:
Would be consequent, IMHO, given you mention the Kit Standard doesn't provide PoE

Reply:
> Would be consequent, IMHO, given you mention the Kit Standard doesn't provide PoE

did you mean "comes with", instead of "does not come with"? Because it does come with a power supply. Or did you want to mention PoE?

Reply:
I'm sorry 🤦 ! The additional sentence belong on line done - to the Yellow Kit PoE!

Reply:
Thank you for the catch. Fixed.